ci/new-pull-request.yml

```diff
@@ -44,7 +44,7 @@ stages:
           - task: Maven@3
             displayName: "Compile with Maven"
             inputs:
-              mavenPomFile: "core/pom.xml"
```

Comment:
I don't think this is used anymore. The file that its used for "New pull request" pipeline is test-pipeline.yml
Probably we need to rename it to avoid confusion.